### PR TITLE
Upgrade to node 16 [POL-142]

### PR DESCRIPTION
**Pull request by zarsky-broad**
Node 16 is the current LTS; additionally, it has native support for apple silicon.

Verified running and tests passing. I updated only dependencies used for testing, since those are run by node, rather than transpiled.

I've also added notifying github about deployments, to try to publicize PR sites more. I'm trying to figure out if it's too noisy.

### CODE REVIEW

File: .circleci/config.yml
Comment (kyuksel) [inline .circleci/config.yml:70]: `env_var_name` is a valid `type`?

Comment (kyuksel) [reply .circleci/config.yml:70]: nevermind - found the [doc](https://circleci.com/docs/2.0/reusing-config/#environment-variable-name) 😊

**Review by kyuksel — APPROVED** [review]
LGTM 👍🏼 
`yarn install`ed and `yarn test`ed.
Thanks for taking the time to do the upgrade 🙏🏼

**Comment by kyuksel** [comment]
Where is the deployment notification supposed to show up?

**Comment by kyuksel** [comment]
Oh and let's make sure to give folks a heads-up on Slack before merging 😊

**Comment by zarsky-broad** [comment]
There's a quick link to a deployment in 3 locations:
1. In the conversation view, following each commit
2. In the conversation view, just above the status summary and the merge button at the bottom
3. On the main page of the repo, in the bottom right corner

**Comment by zarsky-broad** [comment]
@dmohs You might find this one interesting

**Comment by kyuksel** [comment]
Verified that the link below works  🚀 

image

**Review by dmohs — APPROVED** [review]

**Pull request merged**
